command_config.json

```diff
@@ -0,0 +1,26 @@
+{
+    "rand": {
+        "command": "rand",
```

Comment:
Goal: We shouldn't need `command` here

Reply:
yep, not suitable for `issues = [x async for x in get_issues(**command_config)]` anyway